shanoir-ng-import/src/main/java/org/shanoir/ng/importer/dicom/ImagesCreatorAndDicomFileAnalyzerService.java

```diff
@@ -89,10 +94,17 @@ public void createImagesAndAnalyzeDicomFiles(List<Patient> patients, String fold
 				Study study = studiesIt.next();
 				// serie level
 				List<Serie> series = study.getSeries();
+				int nbSeries = series.size();
```

Comment:
Great!!! Here it is correct, as this method is called either after the ZIP file upload (no selection yet) or after the downloadFromPacs. For me not an issue to do the analyze on all series, as very low cost operation in terms of performance. With kind regards, Michael